backend/functions/lib/controllers/case-associated/case-associated.controller.ts

```diff
@@ -11,8 +11,8 @@ const MODULE_NAME = 'CASE-ASSOCIATED-CONTROLLER';
 export class CaseAssociatedController {
   private readonly useCase: CaseAssociatedUseCase;
 
-  constructor(applicationContext: ApplicationContext) {
-    this.useCase = new CaseAssociatedUseCase(applicationContext);
+  constructor(_applicationContext: ApplicationContext) {
```

Comment:
Why not just remove the context argument?
```suggestion
  constructor() {
```